Quick fix to universal drill

vdubikhin opened the pull request:
Fixed problem with universal drill not showing buttons on activation. Problem introduced in https://github.com/sswelm/KSP-Interstellar-Extended/pull/144

sswelm commented:
interesting, so the mass parameter actually has an effect, but a negative one. It's still unclear to me when and how to use it.

vdubikhin commented:
The problem I resolved was not with mass parameter, but animation not finishing playing after part load. In the previous commit I have introduced some logic that checks if deploy animation is playing to show buttons. Apparently, setting normalized time is not enough and you have to set play speed as well: L195 ahd L201.

The second parameter, targetWeight, in deployAnimation.Blend specifies blend distribution between animation layers. The parameter must be in the range [0, 1] and is equal to 1 by default, which means only top layer is used. Using part.mass is not correct as well as you do not have to specify it to be equal to 1.

 That is why I removed those parameters and use default Blend function, which is in fact a wrapper around Blend(animation, targetWeight=1f)
